apps/condo/domains/billing/utils/serverSchema/index.js

```diff
@@ -69,15 +70,37 @@ async function registerBillingReceipts (context, data) {
  * @param period {string}
  * @return {Promise<*>}
  */
-const getPaymentsSum = async (context, organizationId, accountNumber, period, bic, bankAccount) => {
-    const payments = await  find('Payment', {
-        organization: { id: organizationId },
-        accountNumber: accountNumber,
-        period: period,
-        status_in: [PAYMENT_DONE_STATUS, PAYMENT_WITHDRAWN_STATUS],
-        recipientBic: bic,
-        recipientBankAccount: bankAccount,
-    })
+const getPaymentsSum = async (context, receiptId, organizationId, accountNumber, period, bic, bankAccount) => {
+    const directReceiptIdWhere = { AND: [
+        { receipt: { id: receiptId } },
+    ],
+    }
+
+    // TODO(YEgorLu): DOMA-10222 create task to connect receipt to early payment after receipt creation and remove this check
+    const oldWhereByOrganizationCredentials = { AND: [
```

Comment:
it looks too complex

Reply:
it is...

This check almost duplicates old one. It is fields to know, that payment was made from certain user account to certain organization account so we can assume that payment was made for certain receipt and adds check for receiptId = null, to not to interfere with new logic

Will be removed from here after DOMA-10222 (most likely will live in new worker)